bin/aurcheck

```diff
@@ -76,6 +76,19 @@ namever_aur() {
     xargs -r aursearch -ir | jq -er '.results[] | [.Name, .PackageBase, .Version] | @tsv'
 }
 
+namever_vcs() {
+    for pkg in "$@"; do
+        base=$(aursearch -ir "$pkg" | jq -er '.results[] | .PackageBase')
+        cd "$HOME/.cache/aursync/$base" || continue   # TODO proper error handling, here and below
```

Comment:
Can probably use `cd_safe` if  we move the `makepkg/util.sh` sourcing.

Reply:
Refference: https://git.archlinux.org/pacman.git/tree/scripts/libmakepkg/util/util.sh.in#n79

Reply:
Ah I posted my question like 20 seconds later than you 😃 Just tested `cd_safe`, it prints "Aborting..." and exits the function early and does not let the for loop continue for other "valid" packages. I'm curious on the general approach though, I feel like it's better to ignore the package and continue the check for the rest of the vcs packages, what do you think?

Reply:
This is `cd_safe`:

```
❯ aurcheck -d maximbaz-aur
/usr/share/makepkg/util/util.sh: line 80: cd: /home/maximbaz/.cache/aursync/alacritty-terminfo-git: No such file or directory
==> ERROR: Failed to change to directory /home/maximbaz/.cache/aursync/alacritty-terminfo-git
    Aborting...
alacritty-git 0.1.0.729.g59b561b-1 -> 0.1.0.730.gb82622e-1
alacritty-terminfo-git (not available)
antigen-git (not available)
aurutils-git (not available)
dragon-git (not available)
gtk-theme-arc-grey-git (not available)
i3ipc-python-git (not available)
libva-git (not available)
libva-intel-driver-git (not available)
lscolors-git (not available)
pam_u2f-git (not available)
py3status-git (not available)
python-azure-git (not available)
universal-ctags-git (not available)
```

This is `cd ... || continue`:

```
❯ aurcheck -d maximbaz-aur
/usr/bin/aurcheck: line 82: cd: /home/maximbaz/.cache/aursync/alacritty-terminfo-git: No such file or directory
alacritty-git 0.1.0.729.g59b561b-1 -> 0.1.0.730.gb82622e-1
alacritty-terminfo-git (not available)
aurutils-git 1.5.3.r266.gdc2c713-1 -> 1.5.3.r270.g199cc3b-1
universal-ctags-git 0.r5769.66c7915e-1 -> 0.r5771.5e4eca42-1
```